build_scripts/windows/scripts/build.cmd

```diff
@@ -51,35 +58,50 @@ if exist %REPO_ROOT%\privates (
     copy %REPO_ROOT%\privates\*.whl %TEMP_SCRATCH_FOLDER%
 )
 
-::ensure wix is available
+REM ensure wix is available
 if exist %WIX_DIR% (
     echo Using existing Wix at %WIX_DIR%
 )
 if not exist %WIX_DIR% (
     mkdir %WIX_DIR%
     pushd %WIX_DIR%
     echo Downloading Wix.
-    curl -o wix-archive.zip %WIX_DOWNLOAD_URL% -k
+    curl -o wix-archive.zip %WIX_DOWNLOAD_URL%
     unzip -q wix-archive.zip
     if %errorlevel% neq 0 goto ERROR
     del wix-archive.zip
     echo Wix downloaded and extracted successfully.
     popd
 )
 
-::ensure Python is available
+REM ensure Python is available
 if exist %PYTHON_DIR% (
     echo Using existing Python at %PYTHON_DIR%
 )
 if not exist %PYTHON_DIR% (
+    echo Setting up Python and pip
     mkdir %PYTHON_DIR%
     pushd %PYTHON_DIR%
-    echo Downloading Python.
-    curl -o Python389-32.zip %PYTHON_DOWNLOAD_URL% -k
```

Comment:
`-k, --insecure` was introduced by #4523 allowing non-secure HTTPS, which we don't know why either.